Templates/scripts/signin.js

```diff
@@ -0,0 +1,10 @@
+function unhide() {
+    var x = document.getElementById("login-form");
+    if (x.style.display === "none") {
+      x.style.display = "block";
+    } else {
+      x.style.display = "block";
+    }
```

Comment:
What do you think about removing this condition?? since both cases are doing the same thing.
it could be useful if the instruction in the if block is different from the one the else block